spec/models/manageiq/providers/inventory/persister/builder_spec.rb

```diff
@@ -3,10 +3,7 @@
 
 describe ManageIQ::Providers::Inventory::Persister::Builder do
   before :each do
-    @zone = FactoryBot.create(:zone)
-    @ems  = FactoryBot.create(:ems_cloud,
-                               :zone            => @zone,
-                               :network_manager => FactoryBot.create(:ems_network, :zone => @zone))
+    @ems = FactoryBot.create(:ems_cloud, :zone => FactoryBot.create(:zone))
```

Comment:
Can you remove the `, :zone => FactoryBot.create(:zone)` from all of these?  It's already in the base factory.